pkg/operator/starter.go

```diff
@@ -731,6 +731,7 @@ func prepareOauthAPIServerOperator(ctx context.Context, controllerContext *contr
 		csr.ClientCertOption{
 			SecretNamespace: "openshift-oauth-apiserver",
 			SecretName:      "openshift-authenticator-certs",
+			JiraComponent:   "apiserver-auth",
```

Comment:
I bet this is somehow my fault, but I honestly don't remember creating this one.